packages/types/src/primitive/sszTypes.ts

```diff
@@ -54,11 +57,21 @@ export const Wei = UintBn256;
 export const Root = new ByteVectorType(32);
 export const BlobIndex = UintNum64;
 
+export class ExecutionAddressType extends ByteVectorType {
+  constructor() {
+    super(20);
+  }
+  toJson(value: ByteArray): unknown {
+    const string = super.toJson(value) as string;
```

Comment:
should avoid using `string` as a variable name
```suggestion
    const str = super.toJson(value) as string;
```